bundles/AdminBundle/Controller/Admin/UserController.php

```diff
@@ -797,6 +798,13 @@ public function uploadImageAction(Request $request)
             throw $this->createAccessDeniedHttpException('Only admin users are allowed to modify admin users');
         }
 
+        //Check if uploaded file is an image
+        $avatarFile = $request->files->get('Filedata');
+
+        if (!$avatarFile instanceof UploadedFile || !str_starts_with($avatarFile->getMimeType(), 'image/')) {
```

Comment:
can we use `getTypeFromMimeMapping()` here and check with "image" type? something like https://github.com/pimcore/pimcore/blob/0c3179c170bd1206e8140b3691e338aef80484f9/bundles/AdminBundle/Controller/Admin/Asset/AssetController.php#L534